addon/components/sortable-group.js

```diff
@@ -18,29 +19,35 @@ export default Component.extend({
     @property items
     @type Ember.NativeArray
   */
-  items: computed(() => { return a(); }),
+  items: computed({
+    get() { return a(); }
```

Comment:
I was under the impression that `computed(function() {})` and `computed({ get() {} })` were equivalent with the new form.

/cc @cibernox